influxdb/config.json

```
@@ -47,6 +48,12 @@
     "ssl": "bool",
     "certfile": "str",
     "keyfile": "str",
+    "envvars": [
+      {
+        "name": "match(^INFLUXDB_)",
```

We should tie this down a little, since well, it could now mess with the environment in other ways.

```suggestion
        "name": "match(^INFLUXDB_([A-Z0-9_])+$)",
```

---

Makes sense, updated.